src/compiler/crystal/tools/playground/server.cr

```diff
@@ -10,7 +10,7 @@ module Crystal::Playground
   class Session
     getter tag : Int32
 
-    def initialize(@ws : HTTP::WebSocket, @session_key : Int32, @port : Int32)
+    def initialize(@ws : HTTP::WebSocket, @session_key : Int32, @port : Int32, @@host : String | Nil)
```

Comment:
This should be an instance variable, not a class variable, right?

Reply:
I first use instance variable ```@host```, but got error ```Error: @instance_vars are not yet allowed in metaclasses: use @@class_vars instead ```, so change to ```@@host```.

Reply:
That's not a proper solution. The error appears because you tried to access the ivar directly from a class method. That doesn't work because it's not bound to an instance. You should pass the ivar value to the class method, just like `@port` (or just combine them into `@address`, which probably makes even more sense).

Reply:
thanks! I had try to modify it by another commit right now!